src/test/java/com/github/awwkoala/selenium/CheckBoxesTests.java

```diff
@@ -16,19 +16,19 @@
   @BeforeClass
   public void prepareTest() {
     cbPage = new CheckboxesPage(driver);
-    cbPage.goToCheckboxesPage();
+    cbPage.goTo();
   }
 
   @Test
   public void checkboxesTest() {
-    cbPage.assertThatCheckbox1NotSelected();
-    cbPage.assertThatCheckbox2Selected();
+    cbPage.assertThatFirstCheckboxNotSelected();
```

Comment:
Its not necessary but you can apply to your test framework something that it's called FluentAPI. https://www.automatetheplanet.com/fluent-page-object-pattern/